portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +26,7 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Aradhya!</h1>");
+		response.setContentType("text/html");
+    response.getWriter().println("Hello Aradhya!");
```

Comment:
Nit: make indentation consistent

Reply:
Got it, don't know how that got indented so weirdly.